.vsts-dotnet-ci.yml

```diff
@@ -52,7 +52,7 @@ jobs:
     pool:
       name: NetCore-Public
       demands: ImageOverride -equals build.ubuntu.1804.amd64.open
-    helixQueue: Ubuntu.1804.ArmArch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm32v7
+    helixQueue: (Ubuntu.2204.Arm64.Open)Ubuntu.2004.ArmArch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm32v7
```

Comment:
I thought that the part in brackets was the display name for the container image. In any case, I see three versions on that one line and am not sure what three things they apply to.